### DISCUSSION TITLE
Orion notifies low battery when plugged and the percentage is actually going up, charging (8%, 10%...)

### DISCUSSION THREAD

**Discussion by yoshi-ortiz** [kagisearch/kagi-docs#198] [Q&A] [answered]
Hi Orion* team,

I would just like to comment that I've been receiving notifications from the Orion app on my MacBook saying that I'm running out of battery when my device is actually changing and plugged in.

I guess this could be considered a **bug**, since this notification **may be only useful when the battery is actually going down**.

Could this behavior be updated?

I would also like to ask... **Can this warning be disabled in any way under my installed version**?
(Orion RC - Sep 16, 2023)

Apart from this, congratulations on your great work!
Orion is such a cool product!

All the best,
Yoshi

**Comment by vprelovac** [comment, marked as answer]
@lord-reptilia Yep that sounds like a bug - please post bug report to https://orionfeedback.org. 

For help/chat with team/community we recommend joining Orion discord.
